graphrag/config/create_graphrag_config.py

```diff
@@ -626,7 +645,7 @@ def hydrate_parallelization_params(
         summarize_descriptions=summarize_descriptions_model,
         umap=umap_model,
         cluster_graph=cluster_graph_model,
-        encoding_model=encoding_model,
+        encoding_model=global_encoding_model,
```

Comment:
shouldn't this use `encoding_model`, which if unset locally has been set to the global var?

Reply:
(the defaulting happens on line 473)

Reply:
Nope. It is a little confusing given how much redundancy is there with encoding config, but this one is the top level one. Line 473 is the local encoding config for claim extraction

Reply:
ok